src/components/home.vue

```diff
@@ -1,21 +1,34 @@
 <template>
-    <div>
-
-    </div>
+  <div class="container">
+    <MovieAndTvShowList
+      :moviesEndpoint="'popular'"
+      :tvShowsEndpoint="'popular'"
+      :title="'Most Popular'"
+    ></MovieAndTvShowList>
+ <MovieAndTvShowList
+      :moviesEndpoint="'now_playing'"
```

Comment:
Please refactor to `moviesEndpoint='now_playing'`. There is no need to use `:` because then you are passing javascript variable or expression.

Reply:
Same applies to while `home.vue` component.